QEfficient/exporter/export_utils.py

```diff
@@ -319,12 +319,12 @@ def run_model_on_ort(
         past_value_mean = past_key_sum / num
         print(f"past_keys (mean) \t\t {past_key_mean}")
         print(f"past_value (mean) \t\t {past_value_mean}")
-        print("\n=============================================================\n")
+        print("\n=====================================================================\n")
```

Comment:
Let's remove print statements and use `logging.info`

Reply:
todo @ochougul